{{cookiecutter.project_slug}}/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,90 @@
+// For format details, see https://containers.dev/implementors/json_reference/
+{
+    "name": "{{cookiecutter.project_slug}}_dev",
+    "dockerComposeFile": [
+        "../local.yml"
+    ],
+    "init": true,
+    "mounts": [
+        {
+            "source": "./.history/bash_history",
+            "target": "/home/dev-user/.bash_history",
+            "type": "bind"
+        },
+        {
+            "source": "~/.ssh",
+            "target": "/tmp",
+            "type": "bind"
+        },
+        {
+            "source": "~/.ssh",
+            "target": "/home/dev-user/.ssh",
+            "type": "bind"
+        }
+    ],
+    // Tells devcontainer.json supporting services / tools whether they should run
+    // /bin/sh -c "while sleep 1000; do :; done" when starting the container instead of the container’s default command
+    "overrideCommand": true,
+    "service": "django",
+    // "remoteEnv": {"PATH": "/home/dev-user/.local/bin:${containerEnv:PATH}"},
+    "remoteUser": "dev-user",
+    "workspaceFolder": "/app",
+    // Set *default* container specific settings.json values on container create.
+    "customizations": {
+        {% if cookiecutter.editor == "VS Code" %}
+        "vscode": {
+            "settings": {
+                "editor.formatOnSave": true,
+                "isort.args": [
+                    "--profile",
+                    "black"
+                ],
```

Comment:
This is partly duplicating [our config](https://github.com/cookiecutter/cookiecutter-django/blob/2e561ed6c4c7ed8950334eaa63eca70a32dac055/%7B%7Bcookiecutter.project_slug%7D%7D/pyproject.toml#L27-L35), and is not the same as what we use (e.g. line length is missing) so I guess that might cause some conflicts between pre-commit and VS Code. 

Do you think we could reuse the project config file instead?